### DISCUSSION TITLE
Always empty array in /history 200 response

### DISCUSSION THREAD

**Discussion by mainefol** [amzn/ads-advanced-tools-docs#243] [Q&A] [answered]
I am requesting data from the /history endpoint. I am getting an empty response back. Here is an example of my request:
"{
"fromDate":1712214434661,
"toDate":1704438434661,
"eventTypes":{
"CAMPAIGN":
{"filters":
 ["BUDGET_AMOUNT",
                "IN_BUDGET",
                "STATUS",
                "END_DATE",
                "START_DATE",
                "PLACEMENT_GROUP",
                "SMART_BIDDING_STRATEGY",
                "DEFAULT_BID_AMOUNT",
                "NAME"],
eventTypeIds; [my ids...]
}}}".
Response that doesn't change no matter what data I change: 
{
    "totalResults": 0,
    "nextToken": null,
    "events": []
}

**Comment by elizafarley** [comment, marked as answer]
Hi @mainefol,

It looks like your `fromDate` and `toDate` might be mixed up. After converting, I see `toDate` is Friday, January 5, 2024 7:07:14.661 AM and `fromDate` is Thursday, April 4, 2024 7:07:14.661 AM. Can you try and reverse those and see if you get a response?

**Reply by mainefol** [reply]
Yeah, its working, thank you very much. How I didn't think of changing these values myself)
